.github/workflows/rust.yml

```diff
@@ -53,6 +53,7 @@ jobs:
       REDIS_URI: redis://localhost:6379
       WEBHOOK_TOKEN: test_webhook_token
       CONFIG_SERVICE_URI: https://config.service.url
+      VPC_TRANSACTION_SERVICE_URI: 'false'
```

Comment:
We should write this down in the `README` – how to do setup the test environment locally.